### ISSUE TITLE
Error in ListAllODocumentsWidget if try packet operations after update by behavior

### ISSUE THREAD

**Issue by SergeyAsm** [OrienteerBAP/Orienteer#512]
This error rised after adding behavior on
 https://github.com/OrienteerBAP/Orienteer/blob/master/orienteer-core/src/main/java/org/orienteer/core/component/widget/browse/ListAllODocumentsWidget.java#L58

**Comment by PhantomYdn** [comment]
Why actually this bahaviour is needed? And what's actual results? Exception?

**Comment by SergeyAsm** [comment]
Yep, exception. Without that behaviour inner list was not refreshed on external "sendActionPerformed()".

**Comment by PhantomYdn** [comment]
Please provide more details: screenshots, screencasts, full stacktrace and etc.
